sycl/plugins/unified_runtime/ur/adapters/hip/command_buffer.cpp

```diff
@@ -0,0 +1,129 @@
+//===--------- command_buffer.cpp - HIP Adapter ---------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===-----------------------------------------------------------------===//
+
+#include "command_buffer.hpp"
+#include "common.hpp"
+
+/// Stub implementations of UR experimental feature command-buffers
+
+UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferCreateExp(
+    ur_context_handle_t, ur_device_handle_t,
+    const ur_exp_command_buffer_desc_t *, ur_exp_command_buffer_handle_t *) {
+  detail::ur::die("Experimental Command-buffer feature is not "
```

Comment:
I was told to use this instead of `ur::die`: https://github.com/intel/llvm/pull/8303/files#r1253525859
Would be good to be consistent on this.

Reply:
`L0` uses a different structure to report errors than `CUDA/HIP` atm. `urPrint` depends on env variables that are defined inside `L0` plugin only atm like `UR_L0_DEBUG`, so to port this functionality across to CUDA/HIP, We will need to generalize this env variables and uses them across all the adapters. So, this is just a temporary solution so that cuda and hip align with each other. And, I guess we could work on generalizing `urPrint` in a different PR as it could be out of context in here to work on redefining those env variables across adapters.

Reply:
OK I see thanks.